### ISSUE TITLE
Q: Custom chords.

### ISSUE THREAD

**Issue by endoveg** [x42/midifilter.lv2#18]
Hi!
I'm working with Midi Chord plugin and I wonder if there is any way I could play, for instance, Asus4 chord, using the plugin.

**Comment by x42** [comment]
You could use 1,5,8,11  (prime, 5th, octave, 11) for that.
Incidentally this is how you would play a sus4 on a guitar.

So far there's no explicit 4th.

**Comment by mstaack** [comment]
@x42 thanks for those lovely midi plugins.
have you thought about extending the chords to more scales? minor ones are missing?

**Comment by x42** [comment]
I'd like to avoid adding redundant scales. That'll soon get out of hand.
https://en.wikipedia.org/wiki/Mode_(music)#Modern_modes

C-major scale is identical to 
 * D Dorian
 * E Phrygian
 * F Lydian
 * G Mixolydian
 * A Aeolian (aka A-minor)
 * B Locrian
 
The plugin offers all triads 1,3,5,7,9,11,13 of a natural Locrian scale that can be used to construct harmonies in the modern 12-TET scale.

**Comment by SpotlightKid** [comment]
The labels of the "Scale" param values may make it hard to find the scale your looking for. E.g. to have the plugin generate the chords of the C-minor, you'll need to use the relative major Eb, but the param value lables lists this only as "D# Major".

Maybe change the labels for accidentals to read e.g. "D#/Eb Major" etc. ?

**Comment by x42** [comment]
> "D#/Eb Major"

Good idea. This will also retain the alphabetical sort order of hosts that sort scalepoints.
I'd have preferred to use flats, but  `Db/C# Major` would be sorted after "D Major".
